README.md

```diff
@@ -99,7 +99,7 @@ npm start
 
```

Comment:
- Fix spelling errors
- add additional context and clarifications
- add docker host networking (so that docker uses the host system as a basis for networking)